src/Webview/Ffi.cs

```diff
@@ -6,16 +6,16 @@ namespace Webview
     [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
     internal delegate void webview_external_invoke_cb_t(
         IntPtr webview,
-        [MarshalAs(UnmanagedType.LPUTF8Str)] string arg);
+        [MarshalAs(UnmanagedType.LPStr)] string arg);
```

Comment:
This change was required because I changed the target framework of the nuget packages from .NET Core to .NET Standard.  .NET Standard does not have UnmanagedType.LPUTF8Str